sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/share_responses.hpp

```diff
@@ -200,10 +200,10 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
 
     /**
      * @brief Get the raw HTTP response.
-     * @return A pointer to #Azure::Core::Http::RawResponse.
+     * @return A #Azure::Core::Http::RawResponse.
      * @note Does not give up ownership of the RawResponse.
      */
-    Azure::Core::Http::RawResponse* GetRawResponse() const override { return m_rawResponse.get(); }
+    Azure::Core::Http::RawResponse& GetRawResponse() const override { return *m_rawResponse.get(); }
```

Comment:
@CaseyCarter can you review this as well. Is this a reasonable pattern to give a reference to a `unique_ptr` to the user while holding on to its ownership?

Reply:
I would expect a const& so it can only be read but not updated

Reply:
+1 for const &